dotCMS/src/main/java/com/dotcms/api/system/event/local/DefaultOrphanEventSubscriber.java

```diff
@@ -0,0 +1,36 @@
+package com.dotcms.api.system.event.local;
+
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.Logger;
+
+/**
+ * The default subscriber to handle the ophan events.
+ * By default it is just doing a log
+ * @author jsanca
+ */
+public class DefaultOrphanEventSubscriber implements EventSubscriber<OrphanEvent> {
+
+    public static final String DEBUG_LEVEL = "debug";
+    public static final String INFO_LEVEL  = "info";
+    public static final String ORPHANEVENTSUBSCRIBER_DEBUGLEVEL = "orphaneventsubscriber.debuglevel";
+    private final String logLevel;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "logLevel" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)